cmd/influxd/backup/backup.go

```diff
@@ -166,5 +323,26 @@ backup downloads a snapshot of a data node and saves it to disk.
         -host <host:port>
                           The host to connect to snapshot.
                           Defaults to 127.0.0.1:8088.
+        -database <name>
```

Comment:
I was following what @benbjohnson had done in his. Maybe this was because of the `PATH` at the end that didn't have any flag associated with it?